account_operating_unit/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": 'Accounting with Operating Units',
+    "summary": "An operating unit (OU) is an organizational entity part of a\
```

Comment:
definition in the README file not in summary.
"Introduces Operating Unit fields in invoices and Accounting Entries with clearing account"